airqo-platform/AirQo-frontend
# [PLATFORM] Platform Account Verification

#### Summary of Changes (What does this PR do?)

- Account creation pages design
- Tailwind colors added
- Redux state management for the user details
- POST to API using API token

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Route to account creation using _'/account/creation'_ to see the initial page
- Using a new email register by following the short steps.
- On the email verification step, check that email for the verification link

#### What are the relevant tickets?

- [AN-369]
- [AN-367]

#### Screenshots 
![image (1)](https://user-images.githubusercontent.com/53012890/235913634-f91f6214-9be9-4a6c-ba2a-113f7b8585ad.png)

![image](https://user-images.githubusercontent.com/53012890/235913424-00f42ddb-6666-4e5a-b491-6b6120f226ca.png)



[AN-369]: https://airqoteam.atlassian.net/browse/AN-369?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
